docs/Getting Started.md

```diff
@@ -0,0 +1,171 @@
+# Getting Started
+Welcome to Inductiva, where large-scale simulations meet simplicity, through a 
+seamless, user-friendly interface that brings cutting-edge computational resources 
+to your fingertips—effortlessly.
+
+This section is designed to provide you with all the necessary information you 
+need to get started with Inductiva API. Here, we've condensed everything so you 
+can install Inductiva, grab usage tips, and find some of its essential features, 
+with ease.
+
+It's all you need to get familiar and make the most out of Inductiva from the 
+get-go.
+
+In this section you will find:
+
+* [Overview]()
+    * [What is Inductiva?]()
+    * [Who is it for?]()
+    * [What can you do?]()
+    * [Why choose Inductiva?]()
+* [Prerequisites]()
+* [Installing Inductiva]()
+* [What to read next]()
+* [Troubleshooting]()
+* [Frequently Asked Questions (FAQs)]()
+* [Useful Terminology]()
+
+
+## Overview
+### What is Inductiva?
+Inductiva is a state-of-the-art Python package that simplifies running large-scale 
+physical system simulations. Just two lines of code unleash its potentional, freeing 
+you from local hardware limits and the technical hassles of scaling simulations. 
+It's all about making big computational tasks a breeze.
+
+### Who is it for?
+Inductiva is perfect for curious minds in research, science, and engineering who 
```

Comment:
I am not sure this is about "curious minds" :)

Inductiva helps scientist and engineers scaling ....